doc/input-output-reference/src/input-for-output.tex

```diff
@@ -1751,43 +1751,43 @@ \subsubsection{Outputs}\label{outputs-029}
 \item
   Zone,Meter,NaturalGasEmissions:Nuclear Low {[}m3{]}
 \item
-  Zone,Meter,ElectricEmissions:Source {[}J{]}
+  Zone,Meter,ElectricityEmissions:Source {[}J{]}
 \item
-  Zone,Meter,ElectricEmissions:CO2 {[}kg{]}
+  Zone,Meter,ElectricityEmissions:CO2 {[}kg{]}
 \item
-  Zone,Meter,ElectricEmissions:CO {[}kg{]}
+  Zone,Meter,ElectricityEmissions:CO {[}kg{]}
 \item
-  Zone,Meter,ElectricEmissions:CH4 {[}kg{]}
+  Zone,Meter,ElectricityEmissions:CH4 {[}kg{]}
 \item
-  Zone,Meter,ElectricEmissions:NOx {[}kg{]}
+  Zone,Meter,ElectricityEmissions:NOx {[}kg{]}
 \item
-  Zone,Meter,ElectricEmissions:N2O {[}kg{]}
+  Zone,Meter,ElectricityEmissions:N2O {[}kg{]}
 \item
-  Zone,Meter,ElectricEmissions:SO2 {[}kg{]}
+  Zone,Meter,ElectricityEmissions:SO2 {[}kg{]}
 \item
-  Zone,Meter,ElectricEmissions:PM {[}kg{]}
+  Zone,Meter,ElectricityEmissions:PM {[}kg{]}
 \item
-  Zone,Meter,ElectricEmissions:PM10 {[}kg{]}
+  Zone,Meter,ElectricityEmissions:PM10 {[}kg{]}
 \item
-  Zone,Meter,ElectricEmissions:PM2.5 {[}kg{]}
+  Zone,Meter,ElectricityEmissions:PM2.5 {[}kg{]}
 \item
-  Zone,Meter,ElectricEmissions:NH3 {[}kg{]}
+  Zone,Meter,ElectricityEmissions:NH3 {[}kg{]}
 \item
-  Zone,Meter,ElectricEmissions:NMVOC {[}kg{]}
+  Zone,Meter,ElectricityEmissions:NMVOC {[}kg{]}
 \item
-  Zone,Meter,ElectricEmissions:Hg {[}kg{]}
+  Zone,Meter,ElectricityEmissions:Hg {[}kg{]}
 \item
-  Zone,Meter,ElectricEmissions:Pb {[}kg{]}
+  Zone,Meter,ElectricityEmissions:Pb {[}kg{]}
 \item
-  Zone,Meter,ElectricEmissions:WaterEnvironmentalFactors {[}L{]}
+  Zone,Meter,ElectricityEmissions:WaterEnvironmentalFactors {[}L{]}
 \item
-  Zone,Meter,ElectricEmissions:Nuclear High {[}kg{]}
+  Zone,Meter,ElectricityEmissions:Nuclear High {[}kg{]}
 \item
-  Zone,Meter,ElectricEmissions:Nuclear Low {[}m3{]}
+  Zone,Meter,ElectricityEmissions:Nuclear Low {[}m3{]}
```

Comment:
Hmm, do we really want a resource type with a space in it? "Nuclear High" and "Nuclear Low" have been there a long time. so I guess it works, but those may be the only meter name components with a space in them. I realize if we change that here, then it crosses into all of the end-use types for these two resources. There's also "Carbon Equivalent". If there's agreement to change these, it could be done as a separate issue.

Reply:
I agree this looked weird, but I wasn't going to make that kind of change here.  Feel free to open up discussion on a separate issue.

Reply:
See #9207.